getting-started/background/translate.go

```diff
@@ -80,6 +85,40 @@ func Translate(ctx context.Context, m PubSubMessage) error {
 			return fmt.Errorf("firestore.NewClient: %v", err)
 		}
 	}
+	return nil
+}
+
+// [END getting_started_background_translate_init]
+
+// [START getting_started_background_translate_string]
+
+// translateString translates text to lang, returning the translated text,
+// the automatically detected source language, and an error.
+func translateString(ctx context.Context, text string, lang string) (translated string, originalLang string, err error) {
+	l, err := language.Parse(lang)
+	if err != nil {
+		return "", "", fmt.Errorf("language.Parse: %v", err)
+	}
+
+	outs, err := translateClient.Translate(ctx, []string{text}, l, nil)
+	if err != nil {
+		return "", "", fmt.Errorf("Translate: %v", err)
+	}
+
+	if len(outs) < 1 {
+		return "", "", fmt.Errorf("Translate got %d translations, need at least 1", len(outs))
+	}
+
+	return outs[0].Text, outs[0].Source.String(), nil
+}
+
+// [END getting_started_background_translate_string]
+
+// [START getting_started_background_translate]
+
+// Translate translates the given message and stores the result in Firestore.
```

Comment:
Is this a convention to include the function name in the comment? I'd leave it out:
* You never risk that comment and function name diverge
* Comment is one word shorter without losing any information

(I might just be used to seeing comments like `Translate the given message` and my brain is trained to associate that with the function below)

Reply:
Yes, it's standard for comments to start with the thing being described. https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences